code/src/components/app-badge/AppBadge.tsx

```diff
@@ -0,0 +1,24 @@
+import Badge from "@mui/material/Badge";
+import { AppBadgeProps } from "@/components/app-badge/AppBadge.types";
+
+import "@/components/app-badge/AppBadge.scss";
+
+const AppBadge = ({
+  className,
+  isRounded = true,
+  size = "medium",
+  color = "contained",
+  ...props
+}: AppBadgeProps) => {
+  const roundVariant = isRounded ? "" : "spa-badge__no-rounded";
```

Comment:
What to you think about creating some class concat utility, that can join classes correctly across the project? I was inspired by `tailwind` approaches. For example like following:

```jsx
export function cc(...classes: unknown[]) {
  return classes.filter(c => typeof c === "string").join(" ");
}
```

Example usage:

```jsx
className={cc(
    "spa-badge",
     `spa-badge__${variant}`
     isRounded ? "" : "spa-badge__no-rounded",
     className
)}
```

I am not sure about naming, maybe you can propose better name.

You should place this function into global `utils` folder

Reply:
Great idea! Created the utility function and replaced it in AppBadge component